src/main/java/org/fundacionjala/coding/ariel/Kata1.java

```diff
@@ -0,0 +1,25 @@
+package org.fundacionjala.coding.ariel;
+
+
+/**
+ * Kata 1.
+ * Ariel Gonzales.
+ */
+public class Kata1 {
+
+    public static final int CONDITION = 5;
+
+    /**
+     * @param text Any String Text.
+     * @return String.
+     */
+    public String rotReverseWord(String text) {
+        String word = "";
+        String[] texts = text.split(" ");
+        for (String t : texts) {
+            String wordMoreFive = String.join(" ", word, new StringBuilder(t).reverse());
+            word = t.length() >= CONDITION ? wordMoreFive : String.join(" ", word, t);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Convert this usage of the ternary operator to an "if"/"else" structure. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1774)